clarifai/rest/client.py

```diff
@@ -3580,11 +3580,37 @@ def search_inputs(self, query, page=1, per_page=20):
       scores
     """
 
-    resource = "searches/"
+    resource = "users/aspire/apps/%s/inputs/searches/" % CLARIFAI_APP_ID
+    filters = []
+    if (len(query['ands']) == 1):
```

Comment:
should we try catch this if else block? don't feel save to have dictionary field access like this